pkg/injector/init_container_test.go

```diff
@@ -9,12 +9,7 @@ import (
 var _ = Describe("Test volume functions", func() {
 	Context("Test getInitContainerSpec", func() {
 		It("creates volume spec", func() {
-			container := &InitContainer{
-				Name:  "-container-name-",
-				Image: "-init-container-image-",
-			}
-			actual, err := getInitContainerSpec(container)
-			Expect(err).ToNot(HaveOccurred())
+			actual := getInitContainerSpec("-container-name-", "-init-container-image-")
```

Comment:
The use of `InitContainer` is limited and adds marginal value - decided to remove it to reduce verbosity.